docs/_sidebar.md

```diff
@@ -20,4 +20,7 @@
 - Specification
   - [Plugin Specification](/spec/plugin.md)
   - [Dataset Specification](/spec/dataset.md)
+- FAQ
```

Comment:
We need to add guidance on this page for its sub pages.

Reply:
done